btree/btree.c

```diff
@@ -3634,7 +3756,7 @@ void m0_btree_update_credit(const struct m0_btree  *tree,
 {
 	struct m0_be_tx_credit cred = {};
 
-	btree_node_update_credit(tree, &cred);
+	node_rec_update_credit(tree->t_desc->t_root, ksize, vsize, &cred);
 	m0_be_tx_credit_mac(accum, &cred, nr);
```

Comment:
This implementation is okay for fixed format; for other formats when the updated value is of different size than the original value there might be changes to other nodes in the tree so this call might be the same as m0_btree_put_credit().